core/cap-0035.md

```diff
@@ -391,26 +413,14 @@ Possible return values for the `ClawbackOp` are:
 - `CLAWBACK_UNDERFUNDED` if the `from` account does not have sufficient
   available balance of `asset` after accounting for selling liabilities.
 
-#### Create Claimable Balance Operation
-
-This proposal introduces a new failure case in the execution of the
-`CreateClaimableBalanceOp`. If the source account of the operation has a
-trustline to the asset with the `CLAWBACK_ENABLED_FLAG` flag set, the
-operation will fail with `CREATE_CLAIMABLE_BALANCE_CLAWBACK_REQUIRED` if one
-of the `claimants` `destination` is not the issuer account of the asset, or
-if the issuer account is a claimant and the `predicate` is not
-`CLAIM_PREDICATE_UNCONDITIONAL_CLAWBACK`.
-
 #### Clawback Claimable Balance Operation
 
 This proposal introduces the `ClawbackClaimableBalanceOp` operation. The
 `ClawbackClaimableBalanceOp` operation destroys a claimable balance,
 effectively returning the asset to the issuer account, burning it.
 
 The operation will only succeed if the claimable balance has in its

```

Comment:
Nit - "has its" instead of "has in its".

Reply:
38476ed